trac-services/trac-svc-meta/build.gradle

```diff
@@ -43,6 +43,8 @@ dependencies {
     // See the NOTICE file for more information on licensing restrictions
     runtimeOnly project(':sql-drivers')
 
+    runtimeOnly project(':aws-config')
+
```

Comment:
We need to think about how we manage plugin dependencies in general. For now, could you make this one depend on the aws_config build flag as well? It should not be enabled by default.